features/0510-dif-pres-exch-attach/README.md

```diff
@@ -241,7 +314,158 @@ Complete message example:
+`presentation_definition.format`|For JSONLD-based credentials: `ldp_vc` and `ldp_vp`.
+`presentation_definition.format.proof_type`|For JSONLD-based credentials: `Ed25519Signature2018` and `BbsBlsSignatureProof2020`.
+
+### Proof Formats
+
+Supported proof formats:
+
+#### Ed25519Signature2018
+
+**Input Parameters:**
+
+* `presentation_definition.format`: `ldp_vp`
+* `presentation_Definition.format.proof_type`: `Ed25519Signature2018`
+* `options.challenge`: a random string value generated by the Verifier
+* `options.domain`: a string value specified set by the Verifier
+
+**Result:**
+
+A [Verifiable Presentation](https://www.w3.org/TR/vc-data-model/#presentations-0) of type
+[Presentation Submission](https://identity.foundation/presentation-exchange/#presentation-submission) containing the
+credentials requested under the `verifiableCredential` property and a `proof` property of type `Ed25519Signature2018`.
+
+<details><summary>Example</summary>
+
+```json
+{
+    "@context": [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://identity.foundation/presentation-exchange/submission/v1"
+    ],
+    "type": [
+        "VerifiablePresentation",
+        "PresentationSubmission"
+    ],
+    "presentation_submission": {
+        "descriptor_map": [{
+           "id": "citizenship_input",
+           "path": "$.verifiableCredential.[0]"
+        }]
+    },
+    "verifiableCredential": [{
+        "@context": "https://www.w3.org/2018/credentials/v1",
+        "id": "https://eu.com/claims/DriversLicense",
+        "type": [
+            "EUDriversLicense"
+        ],
+        "issuer": "did:foo:123",
+        "issuanceDate": "2010-01-01T19:73:24Z",
+        "credentialSubject": {
+            "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+            "license": {
+            "number": "34DGE352",
+            "dob": "07/13/80"
+          }
+        },
+        "proof": {
+            "type": "RsaSignature2018",
+            "created": "2017-06-18T21:19:10Z",
+            "proofPurpose": "assertionMethod",
+            "verificationMethod": "https://example.edu/issuers/keys/1",
+            "jws": "..."
+        }
+    }],
+    "proof": {
+      "type": "Ed25519Signature2018",
+      "proofPurpose": "assertionMethod",
+      "created": "2017-09-23T20:21:34Z",
+      "verificationMethod": "did:example:123456#key1",
+      "challenge": "2bbgh3dgjg2302d-d2b3gi423d42",
+      "domain": "example.org",
+      "jws": "eyJ0eXAiOiJK...gFWFOEjXk"
+  }
+}
+```
+</details>
+
+#### BbsBlsSignatureProof2020
 
+**Input Parameters**:
+* `presentation_definition.input_descriptors.constraints.limit_disclosure`: `required`
+* `presentation_definition.format`: `ldp_vc`
+* `presentation_Definition.format.proof_type`: `BbsBlsSignatureProof2020`
```

Comment:
The VC ends up with a type of `BbsBlsSignatureProof2020` when it's derived, but I think BBS presentation proofs would be `BbsBlsSignature2020`.